db/migrate/20230423152339_create_claim_reasons.rb

```diff
@@ -0,0 +1,8 @@
+class CreateClaimReasons < ActiveRecord::Migration[7.0]
+  def change
+    create_table :claim_reasons, id: :uuid do |t|
+
+      t.timestamps
+    end
+  end
```

Comment:
do we need a claim reasons table? while it could be usefully to allow creating new claim reasons on the fly, this is unlikely to work given that would also need to be replicated in the other apps we will create.

This also goes against the ValueObject concept we are using to hold the list. Happy to discuss further.

Reply:
removed